src/common/utils/validator.js

```diff
@@ -8,6 +8,22 @@ const genderValues = require('../seeddata/egar_gender_choice.json');
 const { MAX_STRING_LENGTH, MAX_REGISTRATION_LENGTH, MAX_EMAIL_LENGTH, USER_FIRST_NAME_CHARACTER_COUNT, USER_SURNAME_CHARACTER_COUNT } = require('../config/index');
 const logger = require('../../common/utils/logger')(__filename);
 const { airportCodeList } = require('../../common/utils/autocomplete');
+const documenttype = require('../../common/seeddata/egar_saved_people_travel_document_type.json');
+
+const documentTypes = documenttype
```

Comment:
documentTypes seems to be used only inside isValidDocumentTypes, could this be moved inside the function closer to where it's been used?

Reply:
This has been done.